cdxev/__main__.py

```diff
@@ -294,12 +294,14 @@ def create_validation_parser(
     parser.add_argument(
         "--filename-pattern",
         help=(
-            "Regex for validation of file name. "
-            "If no Regex is given the default 'name_version_hash_timestamp.cdx.json' "
-            "or 'bom.json' is used, "
-            "where name, version and timestamp are mandatory and taken from metadata. "
-            "Hash is optional as this is not a required information"
+            "Regex for validation of file name. If not specified, a default regex depending on "
```

Comment:
I think, you could be more precisely here and state that you are sticking to the [naming convention](https://cyclonedx.org/specification/overview/#recognized-file-patterns) of cdx, if the schema-type is `default` and otherwise it is our regex. Or at least adjust this information in the documentation, but as we would like to try to automatically generate the documentation, I would prefer that you adjust the help text to be more verbose.

Reply:
Mh, I have concerns with that.
 
The generated documentation can never replace what we have in `available_commands.md` because stuffing all of that into the command-line help would be madness. 
Actually, there already is a section in the docs on filename validation and your comment made me notice that. I will update it with the changes made in this PR. So, thanks, even if that wasn't what you had in mind :D